README.md

```diff
@@ -1,8 +1,34 @@
-# Rust c-lightning client
+<div align="center">
+  <h1>Rust c-lightning client</h1>
 
-[![Status](https://travis-ci.org/laanwj/rust-clightning-rpc.png?branch=master)](https://travis-ci.org/laanwj/rust-clightning-rpc)
-[![Crates.io](https://img.shields.io/crates/v/clightningrpc.svg)](https://crates.io/crates/clightningrpc)
-[![Docs.rs](https://docs.rs/clightningrpc/badge.svg)](https://docs.rs/clightningrpc/)
+  <p>
```

Comment:
Any reason to use html here instead of markdown ?

Reply:
The only reason is that with HTML I can center the text and make the style of the repo introduction different from the style of it. If you prefer to have the introduction of the readme with the same format, I can reintroduce the markdown again

Reply:
Okay, seems ok to keep it like this to me then.